src/summary/history/allergy-card-level-three-add.component.tsx

```diff
@@ -0,0 +1,338 @@
+import React, { DetailedHTMLProps } from "react";
+import { match } from "react-router";
+import SummaryCard from "../cards/summary-card.component";
+import style from "./allergy-card-level-three.css";
+import {
+  getAllergyAllergenByConceptUuid,
+  getAllergyReaction,
+  savePatientAllergy
+} from "./allergy-intolerance.resource";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import { useCurrentPatient } from "@openmrs/esm-api";
+
+const DRUG_ALLERGEN_CONCEPT: string = "162552AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+const ENVIROMENTAL_ALLERGEN_CONCEPT: string =
+  "162554AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+const FOOD_ALLERGEN_CONCEPT: string = "162553AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+
+export function AllergyCardLevelThreeAdd(props: AllergyCardLevelThreeAddProps) {
```

Comment:
I don't really like this naming convention we've come up with with regards to level one, level two, etc. What do you think about calling this AllergyForm?